docs/debugging.md

```diff
@@ -1,5 +1,11 @@
 # Debugging Guide
 
+## Logo for my platform is not appearing on the Home Page or search results
```

Comment:
Logo missing is not tied to custom data platform anywhere in the docs. This should solve that